intro-to-dev-environment/intro-to-command-line.md

```diff
@@ -0,0 +1,178 @@
+
+### `~/Developer/projects`
+
+- This is a folder where you put all of your Ada projects
+- Make this folder with `$ mkdir ~/Developer/projects` for the first time
+- Navigate from anywhere with `$ cd ~/Developer/projects`
+
+### `~/Developer/classroom`
+
+- This is a folder where you put all of your notes, exercises, scratchwork, classroom activities, homework, etc
+- Make this folder with `$ mkdir ~/Developer/classroom`
+- Pick one of these two popular ways to organize your notes:
+  - Organize notes by week. They will have sub-folders nested in here, such as `~/Developer/classroom/week-1`
+  - Organize notes by topic. Recommended topic folder names are `git`, `cs-fun`, `ruby`, `rails`, `js`, `react`, so it is like `~/Developer/classroom/ruby`
+
+Of course, feel free to not follow these recommendations if you have preferred ways of organizing files that will work for you better.
+
+## Always Run Code with `python3`
+
+This curriculum teaches programming in Python 3. By default, many macOS systems come with Python 2 installed. However, the differences between Python 3 and Python 2 are significant! It will become **crucial** to always execute our Python scripts from the command line and specify Python 3.
+
+We can specify that we want to run specifically Python 3, we will always use this command:
+
+```bash
+python3 some_file.py
+```
+
+...where `some_file.py` is the (path and) name of the Python script we want to run.
+
+If we forget to run our Python scripts with `python3`, there's a chance that the script runs as expected, and a chance that the script breaks or has bugs.
+
+Callout: "A lot of tutorials just use the `python` command to run Python scripts! Why? I'm not convinced that we need to use the `python3` command." One way to check if you need to use the `python3` command is to run two commands: `$ python --version` and `$ python3 --version`. If the output of these two commands differs (namely, `python` points to version 2), then we need to **avoid** using the `python` command.
+
+Callout: There are many methods to make shortcuts for the `python3` command, including terminal/bash aliases, and more. Follow your curiosity!
+
+## Continue Learning
+
+It is crucial for our careers as developers to practice using the terminal, and to increase our skill in it. There are hundreds of more commands to learn.
+
+We can make a commitment to ourselves to discover and learn more and more ways to use the command line, to help our development workflow become more and more robust.
+
+Some commands to learn include:
+
+- `man`
+- `mv`
+- `cat`
+- `echo`
+- `grep`
+
+## Exiting Weird Screens on Terminal
+
+Terminal and some commands bring us to different text editors. If you're ever taken to a terminal screen where:
+
+- arrow keys or key presses don't seem to work as expected
+- there is a `:` on the bottom line
+- `ctrl+c` does not exit this screen
+
+Try using the up and down arrows to scroll, and tapping the key `q` to quit.
```

Comment:
Is this referring to Vim? If so, we might need to explain Vim a LITTLE bit.

Reply:
```suggestion
Try using the up and down arrows to scroll, and tapping the key `q` to quit. If you're ever in this situation, macOS's default text editor, `vim` is at play here!
```

What do you think about this? Deeper?

Reply:
```suggestion
Try using the up and down arrows to scroll, and tapping the `q` or `esc` key to quit.  If you're ever in this situation, one of macOS's default text editors, `[vim](https://en.wikipedia.org/wiki/Vim_(text_editor)` is at play here!

See more vim commands [here](https://devhints.io/vim).
```

Something like that. Nano(ew) is another text editor that comes with MacOS but I'm not sure if that fact is important for the context. https://support.apple.com/guide/terminal/use-command-line-text-editors-apdb02f1133-25af-4c65-8976-159609f99817/mac.

Reply:
```suggestion
Try using the up and down arrows to scroll, and tapping the `q` or `esc` key to quit.  If you're ever in this situation, one of macOS's default text editors, [`vim`](https://en.wikipedia.org/wiki/Vim_(text_editor) is at play here! See more vim commands [with this vim resource](https://devhints.io/vim).
```

Adjusted this for two details @audreyandoy :

1. Your syntax for the vim link wasn't quite correct-- the backticks should be around the link text!
2. It's a preference to not have a link with the text "here" as it isn't quite descriptive (think web accessibility).

Check out my suggestion and lmk if you have questions on those style preferences!

Reply:
oh!! Thank you for reminding me about web accessibility! That suggestion looks great. :)